src/lists/list-item.jsx

```diff
@@ -416,7 +421,13 @@ const ListItem = React.createClass({
       style
     );
 
-    return React.createElement('div', {style: mergedDivStyles}, contentChildren);
+    return (
+      <div
+        style={mergedDivStyles}
+        className={className}>
```

Comment:
What about using the `{...other}`?
People may also want to add additional properties.
So I think that `_createDisabledElement()` should receive `other` as a new parameter.